app/src/main/res/values/strings.xml

```diff
@@ -101,6 +101,8 @@
     <string name="list_screen_rating">Rating: %.1f</string>
     <string name="list_screen_display_only_cctv">Only display parkings with CCTV camera</string>
     <string name="all_parkings_radius">All parkings in a radius of %1$d m</string>
+    <string name="note">Edit your note</string>
```

Comment:
These strings does not respect the convention : ScreenName_Variable

Reply:
Fixed